authors/apps/article/models.py

```diff
@@ -0,0 +1,47 @@
+"""
+class to define the structure of the article
+"""
+
+from django.db import models
+
+# Create your models here.
+
+from authors.apps.article.utils import generate_slug
+from authors.apps.authentication.models import User
+
+
+class Article(models.Model):
+
+    # Model for an article
+    author = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="articles", null=True)
+    title = models.CharField(max_length=255, null=False, blank=False,
+                             error_messages={"required": "Add a title for your article."})
+    description = models.TextField(null=False, blank=False,
+                                   error_messages={"required": "Add a description for your article."})
```

Comment:
Even for structured dictionaries, improve the readability
```python
{
    "required": "Add a description for your article."
}
```

Reply:
noted